go.mod

```diff
@@ -1,10 +1,19 @@
 module knative.dev/net-gateway-api
 
-go 1.16
+go 1.17
+
+replace (
+	k8s.io/api v0.24.1 => k8s.io/api v0.23.8
+	k8s.io/apimachinery v0.24.1 => k8s.io/apimachinery v0.23.8
+	k8s.io/client-go v0.24.1 => k8s.io/client-go v0.23.8
+	k8s.io/code-generator v0.24.1 => k8s.io/code-generator v0.23.8
+	sigs.k8s.io/gateway-api v0.5.0-rc1 => sigs.k8s.io/gateway-api v0.4.0
```

Comment:
Aren't these `replace` lines backwards? (As evidenced by lack of changes to `go.mod`)

Reply:
Likely. I did it both ways just to be sure to attain the most suffering, ended up picking the wrong one evidently. Will fix, thanks for pointing it out.